# python find edges parameters
 
pre_vertical_thre:       15
pre_minium_len:          15
pre_contour_close_thre:  400
interval_overlap_thre:   20
post_short_thre:         20
post_bind_dist_thre:     30
post_merge_dist_thre:    50  # small of this, actually I did similiar step before
post_merge_angle_thre:   20
post_extend_thre:        0  


# c++ pop up cloud parameters
downsample_contour:    false
use_fast_edges_detection:  false
cloud_downsample_rate: 1  #20

plane_cam_dist_thre: 10

